build-config/resources/Ballerina.toml

```
@@ -7,7 +7,7 @@ export=["avro"]
 keywords = ["avro", "serialization", "deserialization", "serdes"]
 repository = "https://github.com/ballerina-platform/module-ballerina-avro"
 license = ["Apache-2.0"]
-distribution = "2201.11.0"
+distribution = "2201.12.0-20250228-201300-8d411a0f"
```

```suggestion
distribution = "2201.12.0"
```

No need for timestamp here